thunderstorm/src/main/frontend/server-info/Button_VersionUpdate.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import {ComponentSync} from '../core/ComponentSync';
+import {ModuleFE_ServerInfo, OnServerInfoUpdatedListener} from './ModuleFE_ServerInfo';
+import './Button_VersionUpdate.scss';
+
+export type Props = {
+	updateVersion?: () => void // in case someone would like a customized updateVersion process
+}
+
+export class Button_VersionUpdate
+	extends ComponentSync<Props>
+	implements OnServerInfoUpdatedListener {
+
+	__onServerInfoUpdated = () => {
+		this.forceUpdate();
+	};
+
+	private updateVersion = async () => {
+		if (this.props.updateVersion) {
+			this.props.updateVersion();
+			return;
+		}
+
+		window.location.reload();
+	};
+
+	render(): React.JSX.Element {
+		const shouldUpdateVersion: boolean = ModuleFE_ServerInfo.Version.shouldUpdateVersion();
+		const newVersion: string = ModuleFE_ServerInfo.Version.getLatestVersion();
+		if (!shouldUpdateVersion)
+			return <></>;
```

Comment:
I think you can return "" instead if <></>

Reply:
Both translate into the same in the end.
This is more meaningful than returning an empty string.
This way, at first glance you understand a div is returned here, rather than the possibility of a string.